tests/video/test_loading_cursor.py

```diff
@@ -0,0 +1,27 @@
+from panda3d.core import MovieVideo
+from panda3d.core import MovieVideoCursor
+import pytest
+
+def get_movie():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = MovieVideo.get(movie_path)
+    return getFile #Obtains a MovieVideo that references a file
+
+@pytest.fixture
+def cursor_check():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = get_movie() #references the function above
+    valid_cursor_check = MovieVideo.open(getFile) #Open this video, returns a MovieVideoCursor 
+
+
+def test_loading():
+    getFile = get_movie()
+    FileReturn = MovieVideo.getFilename(getFile)
+    assert FileReturn == FileReturn
```

Comment:
A variable compared to itself should always be True, so this assert does nothing.

Reply:
I think you might want to do `assert FileReturn.open() is not None`